user-interface/src/components/Stopwatch/index.js

```diff
@@ -12,39 +10,37 @@ import Grid from 'material-ui/Grid';
 import Paper from 'material-ui/Paper';
 import Typography from 'material-ui/Typography';
 
+// Adds leading zeros to a number
+const pad = (number, length, ending) => (`0000${number}`).slice(-length) + ending;
```

Comment:
Consider renaming this function to something like `padZeros` as there are already supported string functions like [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)